api/src/pcapi/core/mails/transactional/educational/eac_finance_incident_notification.py

```diff
@@ -0,0 +1,33 @@
+from pcapi.core import mails
+from pcapi.core.finance import models as finance_models
+from pcapi.core.mails import models as mails_models
+from pcapi.core.mails.transactional.sendinblue_template_ids import TransactionalEmail
+
+
+def send_eac_finance_incident_emails(
+    finance_incident: finance_models.FinanceIncident,
+) -> bool:
+    offer = finance_incident.booking_finance_incidents[0].collectiveBooking.collectiveStock.collectiveOffer
+    venue = finance_incident.venue
+
+    if not venue.current_reimbursement_point or not venue.current_reimbursement_point.bookingEmail:
+        return True
+
+    offers = []
+    for booking_finance_incident in finance_incident.booking_finance_incidents:
+        offer = booking_finance_incident.booking.stock.offer
+        if offer not in offers and not booking_finance_incident.is_partial:
+            offers.append(offer)
+
+    for offer in offers:
+        data = mails_models.TransactionalEmailData(
+            template=TransactionalEmail.RETRIEVE_INCIDENT_AMOUNT_ON_COLLECTIVE_BOOKINGS.value,
+            params={"OFFER_NAME": offer.name, "VENUE_NAME": venue.common_name},
+        )
+
+        mails.send(
+            recipients=[venue.current_reimbursement_point.bookingEmail],
+            data=data,
+        )
+
+    return True
```

Comment:
[discussion] Les 2 fonctions d'envoi de mail se ressemblent beaucoup. A-t-on vraiment besoin de 2 fonctions distinctes ? Est-ce qu'une seule fonction, avec quelques `if` bien placés, ne serait pas suffisamment plus lisible ? Je tente : 

```python
def send_individual_finance_incident_emails(
    finance_incident: finance_models.FinanceIncident,
) -> bool:
    venue = finance_incident.venue

    if not venue.current_reimbursement_point or not venue.current_reimbursement_point.bookingEmail:
        return True

    offers = set()
    for booking_finance_incident in finance_incident.booking_finance_incidents:
        # The mail template refers to a cancelled event: it can only
        # be used for full cancellation, not for a "partial" incident.
        if not booking_finance_incident.is_partial:
            continue
        booking = booking_finance_incident.booking or booking_finance_incident.collectiveBooking
        if isinstance(booking, bookings_models.Booking):
            offer = booking.stock.offer
        else:
            offer = booking.collectiveStock.collectiveOffer
        offers.add(offer)

    for offer in offers:
        if isinstance(offer, offers_models.Offer):
            template = TransactionalEmail.RETRIEVE_INCIDENT_AMOUNT_ON_INDIVIDUAL_BOOKINGS
        else:
            template = TransactionalEmail.RETRIEVE_INCIDENT_AMOUNT_ON_COLLECTIVE_BOOKINGS
        data = mails_models.TransactionalEmailData(
            template=template.value,
            params={"OFFER_NAME": offer.name, "VENUE_NAME": venue.common_name},
        )
        mails.send(
            recipients=[venue.current_reimbursement_point.bookingEmail],
            data=data,
        )

    return True
```

Ça me semble assez lisible (pas de `if`), et plus concis (moins de copié-collé) que 2 fonctions. Qu'en penses-tu ?

Au passage, j'ai changé `offers` en un `set`, ce qui évite un `if`.